draft-ietf-quic-recovery.md

```diff
@@ -127,15 +127,17 @@ these protocol differences below.
 ### Separate Packet Number Spaces
 
 Because QUIC packets must be received and processed before being acknowledged,
-it uses separate packet number spaces for each encryption level.  Separating the
+QUIC uses separate packet number spaces for each encryption level
+(except that 0-RTT and all generations of 1-RTT keys use the same
```

Comment:
Maybe remove the parentheses and simply say, "..for each encryption level, with the exception that 0-RTT..."